### PR TITLE
Add integration test for Search Relevance Dashboards

### PR DESCRIPTION
**Pull request by mingkunm**
Signed-off-by: Mingkun Ma <mingkunm@amazon.com>

### Description

Add integration test for Search Relevance Dashboards 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by tianleh** [comment]
@mingkunm 

1. Please add a workflow for this plugin. See this example for reference. https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/.github/workflows/anomaly-detection-release-e2e-workflow.yml

2. Please let us know the backport branches so that we can add the labels for you.

**Comment by tianleh** [comment]
The `main` CI is running against latest 3.0.0. It is currently failing https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/3380752972/jobs/5614078086

Could you double check if your changes have been built into 3.0.0 bundle or there is some issue with the test case itself?

**Comment by mingkunm** [comment]
The failing test case is assuming [opensearch_dashboards_sample_data_ecommerce](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/5cbaf2ed5af61df15e76b2bce0226ebb4c951868/cypress/integration/common/dashboard_sample_data_spec.js#L200) index has been installed on dashboard.

@tianleh What change should I make if the index hasn't been installed at this point? Thanks!

**Comment by tianleh** [comment]
> 

https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/5cbaf2ed5af61df15e76b2bce0226ebb4c951868/cypress/integration/common/dashboard_sample_data_spec.js#L104

> The failing test case is assuming [opensearch_dashboards_sample_data_ecommerce](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/5cbaf2ed5af61df15e76b2bce0226ebb4c951868/cypress/integration/common/dashboard_sample_data_spec.js#L200) index has been installed on dashboard.
> 
> @tianleh What change should I make if the index hasn't been installed at this point? Thanks!

Check this example https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/5cbaf2ed5af61df15e76b2bce0226ebb4c951868/cypress/integration/common/dashboard_sample_data_spec.js#L104

**Comment by tianleh** [comment]
The new run still failed. https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/3381698854/jobs/5615948741

File: test_finder.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ OSD_COMPONENT_TEST_MAP=( "OpenSearch-Dashboards:opensearch-dashboards"
                          "securityDashboards:security"
                          "notificationsDashboards:notifications-dashboards"
                          "customImportMapDashboards:custom-import-map-dashboards"
+                         "searchRelevanceDashboards:search-relevance-dashboards"
```
Comment (mingkunm) [inline test_finder.sh:21]: I use [searchRelevanceDashboards](https://github.com/opensearch-project/dashboards-search-relevance/blob/ede673f031f9d431589a97a33ceaa7a5741ed397/opensearch_dashboards.json#L2) as left part and use `cypress/integration/plugins/search-relevance-dashboards/` folder name as right part

@tianleh Can you help me verify if it is correct usage? Thanks!

Comment (tianleh) [reply test_finder.sh:21]: This is correct. The left side is the name in infra's build repo https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.4.0/opensearch-dashboards-2.4.0.yml#L55 The right side is the name of the file folder.

**Review by peterzhuamazon — APPROVED** [review]

**Review by tianleh — APPROVED** [review]

**Pull request merged**